### PR TITLE
created admin product list UI and create  product

### PR DESCRIPTION
**Pull request by balu-9491**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -12,6 +12,10 @@ const routes: Routes = [
     path: 'dashboard',
     loadChildren: () => import('./views/user-views/dashboard/dashboard.module').then(m => m.DashboardModule)
   },
+  {
+    path: 'admin-products-list',
```
Comment (Nithyam1330) [inline src/app/app-routing.module.ts]: It should be admin/product-list

File: src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html [hunk 2]
```diff
@@ -1 +1,56 @@
...
+        <div class="col-md-6 col-lg-6 col-sm-12 col-xs-12">
+          <div class="form-group">
+            <label for="exampleInputEmail1">Email address</label>
+            <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
+          </div>
+        </div>
+        <div class="col-md-6 col-lg-6 col-sm-12 col-xs-12">
+          <div class="form-group">
+            <label for="exampleInputEmail1">Email address</label>
+            <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
```
Comment (Nithyam1330) [inline src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html]: Remove all aria-describedby attributes

File: src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html [hunk 3]
```diff
@@ -1 +1,56 @@
...
```
Comment (Nithyam1330) [inline src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html]: Remove all aria-describedby attributes

File: src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html [hunk 4]
```diff
@@ -1 +1,56 @@
...
+          </div>
+        </div>
+      </div>
+
+
+      <div class="row">
+        <div class="col-md-6 col-lg-6 col-sm-12 col-xs-12">
+          <div class="form-group">
+            <label for="exampleInputEmail1">Email address</label>
+            <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
```
Comment (Nithyam1330) [inline src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html]: Remove all aria-describedby attributes

File: src/app/views/admin-views/admin-products-list/admin-products-list.component.html [hunk 5]
```diff
@@ -1 +1,47 @@
...
+          </tr>
+      </tbody>
+    </table>
+  </div>
+  <div class="d-flex justify-content-between p-2">
+    <ngb-pagination [collectionSize]="70" [(page)]="page" [boundaryLinks]="true"></ngb-pagination>
+  <pre>Current page: {{page}}</pre>
+
+    <select class="custom-select" style="width: auto" [(ngModel)]="pageSize">
+      <option [ngValue]="2">2 items per page</option>
```
Comment (Nithyam1330) [inline src/app/views/admin-views/admin-products-list/admin-products-list.component.html]: Make options as array of page labels

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (balu-9491) [reply src/app/app-routing.module.ts]: pending

Comment (balu-9491) [reply src/app/app-routing.module.ts]: solved problem

File: src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html (hunk 2 above)
Comment (balu-9491) [reply src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html]: solved

File: src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html (hunk 3 above)
Comment (balu-9491) [reply src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html]: solved

File: src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html (hunk 4 above)
Comment (balu-9491) [reply src/app/views/admin-views/admin-products-list/admin-modify-product/admin-modify-product.component.html]: solved

**Pull request closed without merging**